pkg/dogstatsd/server.go

```diff
@@ -289,6 +298,42 @@ func NewServer(serverless bool) *Server {
 		}
 	}
 
+	// Creating a new logger
+	cfg := seelogCfg.NewSeelogConfig(
+		"dogstatsd",
+		"info",
+		"common",
+		"",
+		"%Date %Time | %Msg %n",
+		false)
+
+	// Configuring the log file path
+	logFile := config.Datadog.GetString("dogstatsd_log_file")
+	if logFile == "" {
+		logFile = constants.DefaultDogstatsDLogFile
+	}
+
+	// Configuring max roll for log file, if dogstatsd_log_file_max_rolls env var is not set (or set improperly ) within datadog.yaml then default value is 3
+	dogstatsd_log_file_max_rolls := config.Datadog.GetInt("dogstatsd_log_file_max_rolls")
+	if dogstatsd_log_file_max_rolls < 1 {
+		dogstatsd_log_file_max_rolls = 3
+		log.Warnf("Invalid value for dogstatsd_log_file_max_rolls, please make sure the value is higher than 0")
+	}
+
+	// Configure the logger
+	cfg.EnableFileLogging(logFile, config.Datadog.GetSizeInBytes("dogstatsd_log_file_max_size"), uint(dogstatsd_log_file_max_rolls))
```

Comment:
Looking at `SetupJMXLogger` in `config/log.go`, it seems to be busy with the much the same task. Would it make sense to move this code to `config/log.go`? It would allow to keep `seelog_config` internal to config package.